eras/alonzo/impl/cardano-ledger-alonzo.cabal

```diff
@@ -67,7 +67,7 @@ library
         cardano-ledger-allegra >=1.1,
         cardano-crypto-class,
         cardano-ledger-binary >=1.0.1,
-        cardano-ledger-core >=1.6 && <1.7,
+        cardano-ledger-core >=1.6 && <1.8,
```

Comment:
We need to bump the version of the package to `1.4.1.1`

Reply:
Sorry, haven't noticed the branch is out of date. The problem will get resolved by rebasing